styles/style.css

```diff
@@ -1,6 +1,7 @@
+@import url("https://fonts.googleapis.com/css?family=Montserrat");
```

Comment:
Great font ! ,  I am sure we are familiar with this style and decide it to implement it in our design